quinn-proto/src/connection/mod.rs

```diff
@@ -3989,30 +4060,21 @@ impl Connection {
             .push_back(EndpointEventInner::NeedIdentifiers(PathId(0), now, n));
     }
 
-    /// Issues an initial set of CIDs to the peer starting from the next available [`PathId`] in
-    /// use up to the maximum
+    /// Issues an initial set of CIDs for paths that have not yet had any issued
```

Comment:
```suggestion
    /// Issues an initial set of CIDs for paths that have not yet had any issued CIDs
```

not sure it the last word missing is intentional but without it it sounds a bit incomplete to me

Reply:
It was intentional, I'm pretty sure it is correct English without it.  Added it anyway if that makes it easier to understand.